application/src/main/java/org/togetherjava/tjbot/features/mediaonly/MediaOnlyChannelListener.java

```diff
@@ -45,15 +46,10 @@ public void onMessageReceived(MessageReceivedEvent event) {
         }
 
         if (messageHasNoMediaAttached(message)) {
-            message.delete()
-                .flatMap(any -> message.getAuthor()
-                    .openPrivateChannel()
-                    .flatMap(channel -> channel.sendMessage(createNotificationMessage(message))))
-                .queue(any -> {
-                }, failure -> message.getChannel()
-                    .sendMessage(createNotificationMessage(message))
-                    .queue(notificationMessage -> notificationMessage.delete()
-                        .queueAfter(1, TimeUnit.MINUTES)));
+            message.delete().flatMap(any -> dmUser(message)).queue(any -> {
+            }, failure -> tempNotifyUserInChannel(message)
+                .queue(notificationMessage -> notificationMessage.delete()
+                    .queueAfter(1, TimeUnit.MINUTES)));
```

Comment:
please move the details of the temp thing into the method as well. it should look like
```java
message.delete()
  .flatMap(any -> dmUser(message))
  .queue(any -> {}, failure -> tempNotifyUserInChannel(message));
```

Reply:
Just watch the bot now is it ok?
I did update it

Reply:
I this fixed?